app/models/candidate.rb

```diff
@@ -4,17 +4,15 @@ class Candidate < ActiveRecord::Base
   belongs_to :profile
   belongs_to :post
 
-  validates :profile_id, uniqueness: {
-    scope: [:post_id, :election_id], message: 'har redan en likadan kandidatur'
-  }, on: :create
-  validates :name, :lastname, :stil_id, :email,
-            :phone, :post, :profile, :election, presence: true
+  validates :profile_id, uniqueness: { scope: [:post_id,:election_id],message: "har redan en likadan kandidatur"},  on: :create
```

Comment:
Line is too long. [127/100]<br>Unnecessary spacing detected.<br>Space missing after comma.<br>Space inside } missing.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.